extensions/applicationinsights-react-js/package.json

```diff
@@ -53,7 +53,7 @@
         "ts-jest": "^27.0.7",
         "typescript": "^4.3.4",
         "tslib": "^2.0.0",
-        "uglify-js": "^3.11.0",
+        "uglify-js": "3.16.0",
```

Comment:
Pinning to the version of uglify-js that has been tested with this work-around, so that a future change doesn't introduce another issue.

This pinning will be removed for the beta, as we are dropping ES3 (IE8) support.